### PR TITLE
  add springboot-interop-aws-fips for ocp 4.17

### PR DESCRIPTION
**Pull request by amp-rh**
add springboot-interop-aws-fips for ocp 4.17

### CODE REVIEW

**Comment by amp-rh** [comment]
/pj-rehearse periodic-ci-rhoar-qe-spring-boot-openshift-interop-tests-main-springboot-ocp4.17-lp-interop-springboot-interop-aws-fips

File: ci-operator/config/rhoar-qe/spring-boot-openshift-interop-tests/rhoar-qe-spring-boot-openshift-interop-tests-main__springboot-ocp4.17-lp-interop.yaml [hunk 1]
```diff
@@ -49,6 +49,32 @@ tests:
...
+        {
+          "failure_rules":
+            [
+              {"step": "springboot-prepare-cluster", "failure_type": "all", "classification":  "Infrastructure Provisioning - Cluster Preparation", "group": {"name": "lp-tests", "priority": 1}, "jira_additional_labels": ["!default","interop-tests"]},
+              {"step": "springboot-tests-interop", "failure_type": "pod_failure", "classification": "Test Execution", "group": {"name": "lp-tests", "priority": 2}, "jira_additional_labels": ["!default","interop-tests"]},
+              {"step": "springboot-tests-interop", "failure_type": "test_failure", "classification": "Test Failure", "jira_project": "ENTSBT", "group": {"name": "lp-tests", "priority": 2}, "jira_additional_labels": ["!default","interop-tests"]}
+            ]
+        }
+      FIREWATCH_CONFIG_FILE_PATH: https://raw.githubusercontent.com/CSPI-QE/cspi-utils/main/firewatch-base-configs/aws-ipi/lp-interop.json
+      FIREWATCH_DEFAULT_JIRA_ADDITIONAL_LABELS: '["4.17-lp","self-managed-lp","springboot-lp"]'
```
Comment (dfrazzette) [inline ci-operator/config/rhoar-qe/spring-boot-openshift-interop-tests/rhoar-qe-spring-boot-openshift-interop-tests-main__springboot-ocp4.17-lp-interop.yaml:69]: @amp-rh - Requires `"fips"` in `FIREWATCH_DEFAULT_JIRA_ADDITIONAL_LABELS` array.

**Comment by amp-rh** [comment]
/pj-rehearse periodic-ci-rhoar-qe-spring-boot-openshift-interop-tests-main-springboot-ocp4.17-lp-interop-springboot-interop-aws-fips

**Pull request merged**